### DISCUSSION TITLE
Can We GitHub Sponsor You?

### DISCUSSION THREAD

**Discussion by nicholas-stevenson** [cmcpasserby/MayaCharm#66] [General]
Hey Chris,

I just wanted to say thank you so much for all of your work on MayaCharm.  It's a tool that I use every single day, and this realizing hits me every time PyCharm hops a version and MayaCharm isn't yet available.  I also see others asking on Jetbrain's plugin page every time it stops working with the latest PyCharm.  It is such a pleasant workflow between PyCharm and Maya, and I can't imagine having to work without it.  

I just wanted to say that, if you ever chose to enable GitHub sponsoring, I would be first in line to offer something back.  I personally find this tool completely indispensable, and it would be great to be able to do something nice for you in thanks for all your work.

So, I just wanted to offer my thanks, and to express interest if you ever chose to enable that on your GitHub account.  Thank you again Chris!

https://github.com/sponsors

**Comment by cmcpasserby** [comment]
So the version 3.2.6 that I released 2 days ago, should be working right now, and not be prone to breaking on minor versions anymore.

The issue is with continuing development and better supporting the plugin is that I no longer use Maya for my job and thus no longer have access to a up to date version of Maya. So that makes testing anything but build issues and UI issues very hard now. Since its now subscription based it costs $2215 CAD per year which puts it higher then I would consider acceptable for just swallowing that cost of development.

I would not want to be taking in money for it unless I knew that would be enough to cover costs, and feel I like with something like GitHub sponsors it would be very easy to situation where only small amount gets covered putting me on the hook to cover the rest so I can deliver something for those that paid.

Though I am glad that so many people like the tool and find it useful, its just not easiest thing to support at the moment.

**Comment by nicholas-stevenson** [comment]
Aww!  I figured it was something that had pulled you away.  I, and I'm sure everyone else, completely understand.  I even felt a bit nervous even asking since, as you described, the invisible pressure to now maintain something that people were helping contribute and donating towards, would immediately creep in.  But I only asked out of sincere appreciation for you creating this and thanklessly maintaining it.  

Thank you again Chris, this tool has been wonderful to have, and I'm sure everyone really appreciates it!

**Comment by cmcpasserby** [comment]
Yeah ideally i need to spend sometime documenting where everything is, and trying to make it easier for people to contribute to it code wise. Really nothing of what the plugin does is complicated it just takes a while to get a handle on the API pycharm gives you to work with.
